src/main/java/org/jabref/model/entry/ParsedFileField.java

```diff
@@ -1,7 +1,14 @@
 package org.jabref.model.entry;
 
+import java.io.File;
 import java.net.URL;
+import java.nio.file.Path;
 import java.util.Objects;
+import java.util.Optional;
+
+import org.jabref.logic.util.io.FileUtil;
```

Comment:
May I also put `FileUtil` in `model` since it is our abstraction of the `File` object? Reason: I really want the `ParsedFileField` to have a method that returns the path to the linked file. There are around 15 places where `expandFilename` is used upon `ParsedFileField::getLink`.